app/controllers/email2fa_alert_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Email2faAlertController < ApplicationController
+  def show
+    if current_user
+      @status = Email2faService.new.email_2fa_status(current_user.uid)
+    end
+    @enable_link = email_2fa_enable_url
+    @help_link = ENV["ABOUT_2FA_URL"]
+    @session_flag = session[:email_2fa_flag]
+    render layout: false
+  end
+
+  def dismiss
+    session[:email_2fa_flag] = true
```

Comment:
Just checking if this is meant to always be set to true

Reply:
Yes, this is to indicate that the user has dismissed the alert, so we don't show it again in the same session.